packages/libs/react-components/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,59 @@
+import { SystemIcons } from '../Icons';
+
+import {
+  StyledIconWrapper,
+  StyledInput,
+  StyledInputWrapper,
+  StyledLeadingText,
+} from './styles';
+
+import classnames from 'classnames';
+import React, { FC } from 'react';
+
+export interface IInputProps
+  extends Omit<
+    React.HTMLAttributes<HTMLInputElement>,
+    'as' | 'disabled' | 'children'
+  > {
+  as?: 'input';
+  leftPanel?: typeof SystemIcons[keyof typeof SystemIcons] | string;
+  rightPanel?: typeof SystemIcons[keyof typeof SystemIcons];
+  disabled?: boolean;
+  status?: 'success' | 'error';
+}
+
+export const Input: FC<IInputProps> = ({
```

Comment:
the input field is missing a label and tag.
I see that it is available in the input group.
If this is what was discussed, than I still think the input field should have a label, for a11y sake.
also the optional and helpermessage are per input field, no?

Reply:
yes, following discussion with @eileenmguo and @isaozler we decided to put label, tag, info, and helper text in input group
although all together are also available as a text field